analytics-provider/src/main/java/org/digma/intellij/plugin/analytics/AnalyticsProvider.java

```diff
@@ -89,4 +90,6 @@ public interface AnalyticsProvider extends Closeable {
     PerformanceMetricsResponse getPerformanceMetrics();
 
     DeleteEnvironmentResponse deleteEnvironment(DeleteEnvironmentRequest deleteEnvironmentRequest);
+
+    String getDashboard(Map<String,String> fields);
```

Comment:
what is the siganature, can you doc what is the key and what is the value?

Reply:
added